### DISCUSSION TITLE
Doesn't seem the removal of org.keycloak.services.filters.AbstractRequestFilter has been documented

### DISCUSSION THREAD

**Discussion by travis-mcginley** [keycloak/keycloak#31198] [Q&A]
Hello all,

It seems the removal of `org.keycloak.services.filters.AbstractRequestFilter` has been been documented in the release notes of versions 25.0.0 and 25.0.1.  This removal is causing some breaking changes within a library being used.  Would it make sense to update the documentation / release notes to reflect this change?  Also, I'm interested in hearing any opinions on alternatives for this class.

Thanks!

**Comment by mohdfarhanakram** [comment]
you can use below class instead of AbstractRequestFilter

import org.keycloak.services.filters.KeycloakSecurityHeadersFilter
**KeycloakSecurityHeadersFilter**

https://www.keycloak.org/docs-api/25.0.5/javadocs/org/keycloak/services/filters/KeycloakSecurityHeadersFilter.html
